.circleci/config.yml

```diff
@@ -28,4 +28,4 @@ jobs:
           destination: coverage
       - run:
           name: Publish Coverage
-          command: "bash <(curl -s https://codecov.io/bash) -f target/coverage/codecov.json"
\ No newline at end of file
+          command: "bash <(curl -s https://codecov.io/bash) -f target/coverage/codecov.json"
```

Comment:
Ah, we need to redo this entirely. The bash uploader is deprecated and will be increasingly subject to availability brown-outs.